src/lib/tile-processing/vector/qualifiers/factories/helpers/getTreeHeight.ts

```diff
@@ -0,0 +1,21 @@
+import { parseMeters } from "./tagHelpers";
+
+export default function getTreeHeight(tags: Record<string, string>): number | undefined {
+	const minHeight = parseMeters(tags['min_height']) || 0.0;
+	let height = (parseMeters(tags['height']) || parseMeters(tags['est_height'])) - minHeight;
```

Comment:
height can be NaN if both tags are not present

Reply:
This is actually the reason why I used `||` instead of `??`:

```js
undefined ?? 0 == 0
NaN ?? 0 == NaN

undefined || 0 == 0
NaN || 0 == 0
```

Reply:
In JS `NaN == NaN` is `false`, so your example can be missunderstood. 
```js
undefined ?? 0 // is 0
NaN ?? 0 // is NaN

undefined || 0 // is 0
NaN || 0 // is 0

Reply:
Hmm okay. My point is that if I use `??` the code would probably get more complicated. @StrandedKitty should I still change it to `??` and similar constructs?

Reply:
I think it's fine as is